client/src/layout/menu.js

```diff
@@ -208,6 +208,7 @@ export function fetchMenu(options = {}) {
                 title: _l("Galaxy Version: " + Galaxy.config.version_major),
                 url: versionUserDocumentationUrl,
                 target: "_blank",
+                tooltip: `${options.version_major} ${options.version_minor}`,
```

Comment:
I think the `extra` field might be what @afgane is looking to expose, since that contains some anvil-specific stuff.

Reply:
Yeah, thinking about this more, if it's expected folks will shove release note links/etc in there, then this should be a little modal that pops up.  "About this Galaxy" kind of thing, like 
![image](https://user-images.githubusercontent.com/155398/161612323-d00dacb1-7cb2-4c62-ae2e-52edc89cb5ff.png)

Reply:
A modal with all the info available via `/api/version` sounds ok. 

(An issue in AnVIL specifically is that a user cannot enter an address into the browser because of an additional header value that's requires. Hence, only endpoints that have a clickable link in the Galaxy UI are actually accessible to the end-users, which a popup would qualify.)